File: Submissions/day2 .py
number = int(input("Enter a number: "))
if number % 2 == 0:
    print("The number is even")
else:
    print("The number is odd")


age = int(input("Enter your age: "))
if age >= 18:
    print("you are eligible to vote")
else:
    print("you are not eligible to vote")


num1 = int(input("enter first number"))
num2 = int(input("enter second number: "))
if num1 > num2:
   print("the larger number is :", num1)
elif num2 > num1:
   print("the larger number is:", num2)
else:
    print("both numbers are equal")



correct_password = "python111"
password = input("Enter your password: ")

if password == correct_password:
    print("Login Successful")
else:
    print("Incorrect Password")

File: Submissions/day2.cpp
#include <iostream>
#include <string>
using namespace std;

int main() {
    int choice;
    cout << "Choose option:\n";
    cout << "1. Even or Odd\n";
    cout << "2. Voting Eligibility\n";
    cout << "3. Largest of Two Numbers\n";
    cout << "4. Login Check\n";
    cout << "Enter choice: ";
    cin >> choice;

    if (choice == 1) {
        int n;
        cout << "Enter number: ";
        cin >> n;

        if (n % 2 == 0)
            cout << "Even";
        else
            cout << "Odd";
    }

    else if (choice == 2) {
        int age;
        cout << "Enter age: ";
        cin >> age;

        if (age >= 18)
            cout << "Eligible to vote";
        else
            cout << "Not eligible to vote";
    }

    else if (choice == 3) {
        int a, b;
        cout << "Enter two numbers: ";
        cin >> a >> b;

        if (a > b)
            cout << "Largest: " << a;
        else
            cout << "Largest: " << b;
    }

    else if (choice == 4) {
        string pass;
        cout << "Enter password: ";
        cin >> pass;

        if (pass == "Zero_Theory")
            cout << "Login Successful";
        else
            cout << "Incorrect Password";
    }

    else {
        cout << "Invalid choice";
    }

    return 0;
}
